doc/Contributions.md

```diff
@@ -0,0 +1,63 @@
+Our contributions to the ecosystem.
+
+## Our contributions to third-party opensource projects
+
+### XMPPframework for iOS
+
+https://github.com/robbiehanson/XMPPFramework
+
+* XEP-0363: HTTP File Upload: https://github.com/robbiehanson/XMPPFramework/pull/730
+* XEP-0313: Message Archive Management: https://github.com/robbiehanson/XMPPFramework/pull/733
+* XEP-0030: Service Discovery: https://github.com/robbiehanson/XMPPFramework/pull/736
+* MUC light: https://github.com/robbiehanson/XMPPFramework/pull/750
+* Token-based reconnection: https://github.com/robbiehanson/XMPPFramework/pull/758
+* Revamped README: making people feel like this is a well mantained and up to date framework
+* Created a way to Mock a piece of the framework to improve the way we write tests
+
+### Smack for Android
+
+https://github.com/igniterealtime/Smack
+
+* XEP-0357: Push Notifications: https://github.com/igniterealtime/Smack/pull/83
+* XEP-0191: Blocking Command: https://github.com/igniterealtime/Smack/commit/9554c7be26344baf2412c78b6bdf5fc18f0c5e6a
+* XEP-0313: Message Archive Management: https://github.com/igniterealtime/Smack/commit/189cac072b8cedbcecca8bf2b3c5ec4368ea8cfa
+* XEP-0308: Last Message Correction: https://github.com/igniterealtime/Smack/commit/ca58c65a26034bd2b7c4b7e8c2bc70ab593d2ec7
+* MUC light: https://github.com/igniterealtime/Smack/pull/81
+* Token-based reconnection: https://github.com/igniterealtime/Smack/pull/85
+
+### Movim
+
+https://github.com/movim/movim
+
+* Docker image for Movim: https://github.com/esl/movim-docker
+
+## Developped and maintained by Erlang Solutions
+
+### escalus
+
+Available on: [github.com/esl/escalus](https://github.com/esl/escalus)
```

Comment:
If we want to present the whole URL then let's be consistent and put `http(s)://` prefix everywhere.